### PR TITLE
Add example on how to use SpectrumList.read with MIRI MRS dataset

### PR DESCRIPTION
**Pull request by ibusko**
This PR needs https://github.com/astropy/specutils/pull/838

it just adds a new notebook that demonstrates the use of `SpectrumList.read` with MIRI MRS multi-file datasets.

### CODE REVIEW

**Review by javerbukh — APPROVED** [review]
Works! I think this should be updated once jdaviz can load SpectrumList objects, i.e. once #707 is merged.

File: notebooks/SpectrumListExample.ipynb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import glob\n",
+    "import tempfile\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "from astropy.utils.data import download_file\n",
+    "\n",
+    "from specutils import SpectrumList"
```
Comment (pllim) [inline notebooks/SpectrumListExample.ipynb:17]: I don't see any viz used here. Should this really be in a notebook? This reads more like a unit test.

Comment (pllim) [reply notebooks/SpectrumListExample.ipynb:17]: Talking about tests, can you add some? Preferably with the smallest working dataset you can find.

Comment (ibusko) [reply notebooks/SpectrumListExample.ipynb:17]: There are tests in PR #843

Comment (pllim) [reply notebooks/SpectrumListExample.ipynb:17]: I am still not convinced this should be in upper level notebook, unless you can tie its usage to one of the viz. But if you move to concepts folder, I have no objection.

Comment (rosteen) [reply notebooks/SpectrumListExample.ipynb]: You'll be able to load a `SpectrumList` directly into Specviz once #843 is merged. Might be worth adding that here?

Comment (pllim) [reply notebooks/SpectrumListExample.ipynb]: Is this proposal below too controversial?

1. Rebase (not git pull) this PR to get rid of the merge commits.
2. Squash all commits here into one.
3. `git cherry-pick` the result hash here into #843 . The viz integration can continue there.
4. Close this PR *without* merge.

Comment (pllim) [reply notebooks/SpectrumListExample.ipynb]: Though after posting the above, I saw that Ivo moved it to concept, so 🤷

Comment (pllim) [reply notebooks/SpectrumListExample.ipynb]: Also, @ibusko , you forgot to clear the notebook outputs.

**Review by pllim — APPROVED** [review]
I am happy now that it is in concept folder. We can integrate viz later, as Ricky said. Thanks!

**Pull request merged**